src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -22,4 +22,11 @@ public CommandResult execute() {
         List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
         return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
     }
+
+    /**
+     *  Answers whether the command types mutates data
```

Comment:
A more precise statement would be 
"Returns true if .... and false if .. . "
This removes all ambiguity.